templates/.rubocop.yml

```diff
@@ -7,50 +7,17 @@ Metrics/LineLength:
 Style/Documentation:
   Enabled: false
 
-Style/NonNilCheck:
-  IncludeSemanticChanges: true
-
-Naming/PredicateName:
-  Exclude:
-    - 'app/serializers/**/*'
-
-Metrics/BlockLength:
-  Exclude:
-  - config/initializers/simple_form_bootstrap.rb
-  - 'Rakefile'
-  - '**/*.rake'
-  - 'spec/**/*.rb'
-
-Naming/FileName:
-  Exclude:
-  - 'Gemfile'
-
 Style/FrozenStringLiteralComment:
   Enabled: false
-  
-Style/FormatStringToken:
-  Enabled: false
```

Comment:
I don't see the value, but we can also keep it if absolutely wanted.

Reply:
this had false positives. that's why we decided to disable it. I am fine with re-enabling it and if there are still issues they should be addressed in the project itself.